source/firstprogram.rst

```diff
@@ -1,402 +1,621 @@
-Writing your First Program
-==========================
+Writing your First Program:

```

Comment:
@YerikSingh Don't put colons in headings, please.

Reply:
@YerikSingh Fix this, please?

Reply:
done